CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ set_if(LINUX       ${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 set_if(BSD         ${SYSNAME_LC} MATCHES "bsd$")
 set_if(MICROSOFT   WIN32 AND (NOT MINGW AND NOT CYGWIN))
 set_if(SYMLINKABLE LINUX OR DARWIN OR BSD OR CYGWIN)
+set_if(GNU         ${CMAKE_SYSTEM_NAME} MATCHES "GNU")
```

Comment:
Sorry, overlooked previously. The above `SYMLINKABLE` symbol should be defined for GNU as well. So, move this line before the definition for `SYMLINKABLE` and add "OR GNU" at the end.

Reply:
Right, done so